src/fuzz_introspector/datatypes/project_profile.py

```diff
@@ -181,27 +181,26 @@ def resolve_coverage_report_link(
         func_name
     ):
 
-        if self.target_lang == "c-cpp":
-            return "%s%s.html#L%d" % (
+        if self.target_lang == "python":
+            return self.profiles[0].resolve_coverage_link(
                 coverage_url,
                 function_source_file,
-                lineno
+                lineno,
+                func_name
             )
-        elif self.target_lang == "python":
+        elif self.target_lang == "jvm":
             return self.profiles[0].resolve_coverage_link(
                 coverage_url,
                 function_source_file,
                 lineno,
                 func_name
             )
-        elif self.target_lang == "jvm":
-            # TODO Add coverage details for jvm
-            pass
-        return "%s%s.html#L%d" % (
-            coverage_url,
-            function_source_file,
-            lineno
-        )
+        else:
```

Comment:
any reason switching from `if self.target_lang == "c-cpp":` to having it in the else branch?

Reply:
for reducing duplicate code.